pom.xml

```diff
@@ -188,11 +188,16 @@
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>structs</artifactId>
     </dependency>
-    <dependency>
+    <dependency> <!-- Credentials test requires scm-api -->
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>scm-api</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency> <!-- Tests require the scm-api-tests jar -->
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>scm-api</artifactId>
-        <classifier>tests</classifier>
-        <scope>test</scope>
+      <classifier>tests</classifier>
```

Comment:
I may be wrong but from what I see we are repeating the `scm-api` dependency twice. The classifier is used to distinguish artifacts built from the same POM but both the artifacts seem identical to me. 

Other than that, I've successfully built this PR on my local machine, looks good.

Reply:
Never mind, the artifacts are different, it hit me on a careful second look.